tests/unit/test_loggingutil.py

```diff
@@ -34,7 +34,7 @@
     set_timestamps,
 )
 
-
+@pytest.mark.skip
```

Comment:
```suggestion
```

Reply:
My current GPG setting breaks this test. :(